### DISCUSSION TITLE
Fixed layout direction when changing wind directions

### DISCUSSION THREAD

**Discussion by mahdif60** [NatLabRockies/floris#668] [Q&A] [answered]
Is there any possible way to keep layout geometry consistent and only rotate the wind direction? Floris always rotates layout direction to 270 while I want a fixed layout and only rotate wind directions.

**Comment by Bartdoekemeijer** [comment, marked as answer]
Hi @mahdif60. This was very recently taken care of in #578. Please update to the latest FLORIS version and you should be set!

**Reply by mahdif60** [reply]
Thank you so much.
